packages/fast-tooling-react/src/form/form/form-item.array.style.ts

```diff
@@ -80,6 +80,10 @@ const styles: ComponentStyles<FormItemArrayClassNameContract, {}> = {
     },
     formItemArray_controlLabel: {
         ...applyLabelStyle(),
+        borderBottom: "1px solid transparent",
```

Comment:
No squiggly line? 😄 
https://stackoverflow.com/questions/17285514/is-it-possible-to-make-a-squiggly-line

Reply:
Hahaha, that's actually pretty cool. We can update it if we feel it needs a little more "oomph" and the inferred meaning of a red squiggly isn't only regulated to spelling errors.